### ISSUE TITLE
Joint Roadmap for Open Science Tools: how can we improve our ecosystem?

### ISSUE THREAD

**Issue by Daniel-Mietchen** [mozilla/global-sprint#285]
**[ Project Contact ]** @Daniel-Mietchen
**[ GitHub Repo ]** https://github.com/OpenScienceRoadmap/mozilla-sprint-2018
**[ Track ]** Openness, Digital Inclusion, Decentralization, 
**[ Location ]** Charlottesville, VA, U.S. Eastern (UTC -4)
**[ Coach ]** @cbaba20

### Description
We aim to improve the open science landscape systemically by bringing together a range of open projects to explore how they can be improved collectively and integrated towards a more interoperable open science ecosystem. Participants include Wikidata, Mozilla, Zotero, Hypothesis, DAT, Jupyter, bioRxiv, ORCID, Crossref, OJS, Meta, OSF, PLOS and others (see [full list](http://jrost.org/participants)).
***

## Want to contribute to this project during #mozsprint?
Join us at the Global Sprint, May 10-11. Leave a comment below if you're interested in contributing to this project during #mozsprint 2018!

**Comment by dwhly** [comment]
Thanks @Daniel-Mietchen for putting this together!  At [Hypothesis](Hypothes.is) we're excited to see the ways that researchers and others want these tools to work together.  Help us chart the path forward!

**Comment by Daniel-Mietchen** [comment]
@acabunoc This is ready to be featured on Mozilla Pulse.

**Comment by HeidiSeibold** [comment]
Will there be an introduction to the sprint and a distribution of tasks? I am unsure how I can contribute in the best possible way.

**Comment by Daniel-Mietchen** [comment]
@HeidiSeibold I just added some info to our README regarding [coordination](https://github.com/OpenScienceRoadmap/mozilla-sprint-2018/blob/master/README.md#coordination) during the sprint:

We will have a [Zoom channel](https://zoom.us/j/944190942) and the [#opensciroadmap](https://webchat.freenode.net/?channels=#opensciroadmap) channel on IRC available throughout the sprint. We will also be monitoring the [@OpenSciRoadmap](https://twitter.com/OpenSciRoadmap) handle and the [#OpenSciRoadmap](https://twitter.com/hashtag/opensciroadmap?src=hash) hashtag on Twitter, [along with related handles and hashtags](https://twitter.com/search?f=tweets&vertical=default&q=OpenSciRoadmap%20OR%20openscience).

We expect the bulk of the activities during the sprint to result in modifications to [our GitHub repo](https://github.com/OpenScienceRoadmap/mozilla-sprint-2018) and [its issue tracker](https://github.com/OpenScienceRoadmap/mozilla-sprint-2018/issues), to the [repo](https://github.com/OpenScienceRoadmap/OpenScienceRoadmap.github.io) and [issue tracker](https://github.com/OpenScienceRoadmap/OpenScienceRoadmap.github.io/issues) for the [JROST website](http://jrost.org) and (with pointers from our respective issues) to issue trackers and other fora used by projects that form part of the open science landscape, particularly those that participate in the current Mozsprint or that participated in past ones.

If these activities lead to questions or insights that may be of interest to the broader open science community and beyond this sprint, they should probably go to [Ask Open Science](https://ask-open-science.org/), as per [issue 6](https://github.com/OpenScienceRoadmap/mozilla-sprint-2018/issues/6).

@acabunoc Anything else we need to do to get this featured on the Pulse?

**Comment by cbaba20** [comment]
Hey  @Daniel-Mietchen,
Please add one more lable "help-wanted" to those issues which needs contributors support. It will understood by probable contributors more clearly.
Also wanted to know if the image(in repo) is your project logo? its not ideal of logo :) what do you think?

**Comment by Daniel-Mietchen** [comment]
@cbaba20 Our repo has been set up for the sprint, so in essence all the tickets there would fall under "help wanted", but I did a bit of triage and added documentation about this label.

Yes, the image is far from being a logo &mdash; the sprint is basically the first thing that our project is doing, so we do not have a logo yet. I set up a [ticket](https://github.com/OpenScienceRoadmap/mozilla-sprint-2018/issues/14) to create one though.
